lib/purchases_flutter.dart

```diff
@@ -80,23 +80,43 @@ class Purchases {
   /// Set this if you would like the RevenueCat SDK to store its preferences in a different
   /// NSUserDefaults suite, otherwise it will use standardUserDefaults.
   /// Default is null, which will make the SDK use standardUserDefaults.
+  ///
+  /// [useAmazon] Android only. Set this to true if you are building the app
+  /// to be distributed in the Amazon Appstore
+  @Deprecated('Use PurchasesConfiguration')
   static Future<void> setup(
     String apiKey, {
     String? appUserId,
     bool observerMode = false,
     String? userDefaultsSuiteName,
     bool useAmazon = false,
-  }) =>
-      _channel.invokeMethod(
-        'setupPurchases',
-        {
-          'apiKey': apiKey,
-          'appUserId': appUserId,
-          'observerMode': observerMode,
-          'userDefaultsSuiteName': userDefaultsSuiteName,
-          'useAmazon': useAmazon
-        },
-      );
+  }) {
+    final configuration = (
+        PurchasesConfiguration(apiKey)
+        ..appUserID = appUserId
```

Comment:
what are the double dots?

Reply:
TIL about [cascade notation](https://www.dartlang.org/guides/language/language-tour#cascade-notation-). That's pretty cool

Reply:
Yes, I didn't feel the need to create a Builder with this notation